react-app/src/App.js

```diff
@@ -1,20 +1,56 @@
-import logo from './logo.svg';
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from "react-router-dom";
+
+import Home from './components/Home'
+import Extract from './components/Extract'
+import Results from './components/Results'
+
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import './stylesheets/Page.css';
+
 
 function App() {
   return (
-    <div className="App">
+    <div className="page">
+      <Router>
+      <div>
+        {/* A <Switch> looks through its children <Route>s and
+            renders the first one that matches the current URL. */}
+        <Switch>
+          <Route path="/extract">
+            <Extract />
+          </Route>
+          <Route path="/results">
+            <Results />
```

Comment:
The app does not compile for me initially because this `Results` component is blank and has no exports, so React gets mad. Can we either delete this route for now (and it will later be added by your work to display the results), or just have `Results` be a simple component that renders just a `div` or something?